### PR TITLE
Log all the test case which are aborting test run. This will come into picture for parallel run.

### PR DESCRIPTION
**Pull request by Faizan2304**

### CODE REVIEW

**Review by singhsarab — APPROVED** [review]

**Comment by Faizan2304** [comment]
The final output will look like this 

image

File: src/Microsoft.TestPlatform.Extensions.BlameDataCollector/BlameLogger.cs [hunk 1]
```diff
@@ -105,14 +106,17 @@ private void TestRunCompleteHandler(object sender, TestRunCompleteEventArgs e)
             this.output.WriteLine(string.Empty, OutputLevel.Information);
 
             // Gets the faulty test case if test aborted
-            var testCaseName = this.GetFaultyTestCase(e);
-            if (testCaseName == string.Empty)
+            var testCaseNames = this.GetFaultyTestCase(e);
```
Comment (Faizan2304) [inline src/Microsoft.TestPlatform.Extensions.BlameDataCollector/BlameLogger.cs]: Fixed

File: src/Microsoft.TestPlatform.Extensions.BlameDataCollector/BlameLogger.cs [hunk 2]
```diff
@@ -105,14 +106,17 @@ private void TestRunCompleteHandler(object sender, TestRunCompleteEventArgs e)
...
+            if (testCaseNames.Count() == 0)
             {
                 return;
             }
 
-            var reason = Resources.Resources.AbortedTestRun + testCaseName;
-            this.output.Error(reason);
+            this.output.Error(Resources.Resources.AbortedTestRun);
```
Comment (Faizan2304) [inline src/Microsoft.TestPlatform.Extensions.BlameDataCollector/BlameLogger.cs]: Fixed

File: src/Microsoft.TestPlatform.Extensions.BlameDataCollector/Resources/Resources.resx [hunk 3]
```diff
@@ -118,6 +118,6 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="AbortedTestRun" xml:space="preserve">
-    <value>The active Test Run was aborted because the host process exited unexpectedly while executing test </value>
+    <value>The active Test Run was aborted because the host process exited unexpectedly while executing following tests:</value>
```
Comment (Faizan2304) [inline src/Microsoft.TestPlatform.Extensions.BlameDataCollector/Resources/Resources.resx]: Fixed

File: test/Microsoft.TestPlatform.Extensions.BlameDataCollector.UnitTests/BlameLoggerTests.cs [hunk 4]
```diff
@@ -108,6 +108,42 @@ public void TestRunCompleteHandlerShouldGetFaultyTestRunIfTestRunAborted()
         }
 
         /// <summary>
+        /// The test run complete handler should get faulty test run if test run aborted for multiple test project.
+        /// </summary>
+        [TestMethod]
+        public void TestRunCompleteHandlerShouldGetFaultyTestRunIfTestRunAbortedForMultipleProjects()
+        {
+            // Initialize
+            var attachmentSet1 = new AttachmentSet(new Uri("test://uri"), "Blame");
```
Comment (Faizan2304) [inline test/Microsoft.TestPlatform.Extensions.BlameDataCollector.UnitTests/BlameLoggerTests.cs]: fixed

**Review by smadala — APPROVED** [review]

**Pull request merged**